src/lib/telldus-accessory.js

```diff
@@ -215,8 +222,9 @@ class TelldusHygrometer extends TelldusAccessory {
   getHumidity(callback, context) {
     this.log('Checking humidity...')
     TDtool.sensor(this.id, this.log).then(s => {
-      this.log(`Found humidity ${s.humidity}%`)
-      callback(null, parseFloat(s.humidity))
+      const isStale = this.isStale(s)
+      this.log(`Found humidity ${s.humidity}% age ${s.age}s stale:${isStale}`)
+      callback(isStale, parseFloat(s.humidity))
```

Comment:
Cool, and thanks for the link! To me, it looks like we could potentially provide an error message as well. Anyhow, this looks fine to me, I just wanted a reference.